resource-week-5.md

```diff
@@ -0,0 +1,15 @@
+
+
+
+###Encapsulation with JS!
+
+Resource Link: http://www.intertech.com/Blog/encapsulation-in-javascript/
+_By: Jason Shapiro_
+
+#####Why we encapsulate data?
+
+Encapsulation includes the idea that the data of an object should not be directly exposed.  Instead, callers that want to achieve a given result are coaxed into proper usage by invoking methods (rather than accessing the data directly).
```

Comment:
Make sure your readers know this is a direct quote and not your own summary of the resource.